stripe/res/layout/card_multiline_widget.xml

```diff
@@ -65,7 +65,6 @@
             android:layout_marginTop="@dimen/stripe_add_card_element_vertical_margin"
             android:layout_marginEnd="@dimen/stripe_add_card_expiry_middle_margin"
             android:layout_weight="1"
-            app:placeholderText="@string/cvc_multiline_helper"
```

Comment:
I think we want to keep this as-is for general purpose, we just want to remove it for payment sheet usage

Reply:
Done